src/python/espressomd/version.pyx

```diff
@@ -0,0 +1,35 @@
+def major():
+    """Prints the major version of Espresso.
+    """
+    return ESPRESSO_VERSION // 100000
```

Comment:
```python
from __future__ import division
```

Reply:
Why? // works in python 2 and 3 and does what I want.

Reply:
Sorry, I didn't know that.